api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/PaymentCycleTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.dhsc.htbhf.claimant.entity;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.assertions.AbstractValidationTest;
+
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+
+import static uk.gov.dhsc.htbhf.assertions.ConstraintViolationAssert.assertThat;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleTestDataFactory.*;
+
+public class PaymentCycleTest extends AbstractValidationTest {
+
+    @Test
+    void shouldValidatePaymentCycleSuccessfully() {
+        //Given
+        var paymentCycle = aValidPaymentCycle();
```

Comment:
We shouldn't be using `var` in these tests as they don't use the object's constructor.

Reply:
Changed to explicit types.